clarity.d.ts

```diff
@@ -57,7 +57,9 @@ interface IConfig {
 
 declare const enum State {
   Loaded,
```

Comment:
Can you please add the new states at the end. This will break the decoding on the client as the mapping to values will change. If you add at the end then they keep the same encoding.

Reply:
Done

---
In reply to: [158272085](https://github.com/Microsoft/clarity-js/pull/45#discussion_r158272085) [](ancestors = 158272085)